packages/e2e/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 0.4.2
 
 * Adds support for Android E2E tests that utilize other @Rule's, like GrantPermissionRule.
+* Fix CocoaPods podspec lint warnings.
```

Comment:
Published version is 0.4.0

Reply:
I believe you still have to update the CHANGELOG and the pubspec version to a new version?

Reply:
So it will just skip from 0.4.0 to 0.4.3?

Reply:
Well, more likely 0.4.2+1

Reply:
Oh I see what you mean. I guess it's ok since it's not published yet